go.mod

```diff
@@ -36,3 +38,7 @@ require (
 go 1.13
 
 replace github.com/pingcap/tidb => github.com/pingcap-incubator/tinysql v0.0.0-20200518090433-a7d00f9e6aa7
+
+replace github.com/jmhodges/levigo => ./levigo
+
+replace github.com/tecbot/gorocksdb => ./gorocksdb
```

Comment:
What are the changes? Just like the go-ycsb part, we need the diff to review them.

Reply:
I've received feedback that the amount of code including go-ycsb is too large, so I've used a submodule instead.

Reply:
But I was commenting at levigo and gorocksdb, not go-ycsb. They should be submitted the same way as go-ycsb.